src/com/xilinx/rapidwright/design/ModuleInst.java

```diff
@@ -713,6 +720,78 @@ public void connect(String portName, int busIndex0, ModuleInst other, String oth
         }
     }
 
+    /**
+     * Connects unconnected SitePinInst inputs on the module instance to either GND
+     * or VCC (some reset input use VCC as an input and use the inverter to drive
+     * GND).
+     */
+    public void tieOffUnconnectedInputs(boolean verbose) {
+        for (Port port : getModule().getPorts()) {
+            if (port.isOutPort())
+                continue;
+
+            for (SitePinInst pin : getCorrespondingPins(port)) {
+                if (pin == null) {
+                    System.err.println("ERROR: Problem encountered with finding corresponding SitePinInst on port "
+                            + getName() + "/" + port.getName() + " skipping...");
+                }
+                if (pin != null && (pin.getNet() == null || pin.getNet().getSource() == null)) {
+                    boolean useVccInverter = pin.getName().contains("RST");
```

Comment:
This seems like an important condition to document the behaviour of.

The name of this boolean is `useVccInverter` but then when it gets passed to `connectToStaticNet()` it becomes its `connectToVcc` argument which does not seem to reconcile, because it doesn't look like this latter method does does any inversion?

Reply:
I've modified the code to clarify and harmonize the usage.